nevergrad/instrumentation/variables.py

```diff
@@ -334,3 +336,46 @@ class Scalar(Array):
     def __init__(self, dtype: Type[Union[float, int]] = float):
         super().__init__(1)
         self.asscalar(dtype=dtype)
+
+
+class Log(Variable):
```

Comment:
@teytaud instrumenting is actually still very difficult with my way of handling it. I want to simplify it quite a bit (and make it more flexible).
This instrumentation makes (at least) 3 choices I would like you to validate, for a variable like Log(0.001, 0.1):
1. we would tend to sample in the middle to start (i.e. 0.01)
2. we would like mutations to act the same at the center and around the boundaries.
3. we should be able to mutate to one extremity in the order of 10 iterations

This leads to:
1. we linearize [0.001, 0.1] to [-x, x] with x a given choice, 0 maps to 0.01
2. to keep the boundary, we use clipping because arctan makes it actually very difficult to reach the boundary (it maps it to infinity, so you never reach it), the farther you are from the center, the more the mutations are skewed
3. by default I map to [-10, 10] so that reaching the boundaries can be done in the order of magnitude of 10 mutations (can't remember the random walk stuff :D). Not sure it's the best default.

Any thoughts?